### PR TITLE
Remove Raven

### PR DESCRIPTION
**Pull request by mxmeinhold**
I didn't remove the 'report this issue' button. It doesn't work in my local environment, but also a lot of the js in my local environment doesn't quite work, so not sure if it's broken.

### CODE REVIEW

**Comment by devinmatte** [comment]
So this only rips out Raven from Python, what about in the Javascript? Is it only the report issue button?

**Comment by mxmeinhold** [comment]
Didn't touch the JS. I very much don't know the state of conditional's js side, so I'm not confident I would be able to safely.

File: conditional/__init__.py [hunk 1]
```diff
@@ -190,8 +189,7 @@ def route_errors(error, user_dict=None):
     return render_template('errors.html',
                            error=error_desc,
                            error_code=code,
-                           event_id=g.sentry_event_id,
-                           public_dsn=sentry.client.get_public_dsn('https'),
```
Comment (devinmatte) [inline conditional/__init__.py:194]: This line being removed makes me think all JS will break as the JS raven client won't know where to send errors.

Comment (mxmeinhold) [reply conditional/__init__.py:194]:
```git:rm-raven  1M 
(venv) !!! 34 16:19 bionicpi@achlys:conditional $ grep -r Raven conditional/
git:rm-raven  1M 
(venv) !!! 35 16:20 bionicpi@achlys:conditional $ grep -r raven conditional/
git:rm-raven  1M 
(venv) !!! 36 16:20 bionicpi@achlys:conditional $ 
```
... how is raven even set up in the js?

Comment (devinmatte) [reply conditional/__init__.py:194]: Check the `frontend` folder
```
frontend/javascript/modules/errorReport.js:import Raven from 'raven-js';
frontend/javascript/app.js:import Raven from 'raven-js';
```
```
frontend/javascript/modules/errorReport.js:import Raven from 'raven-js';
frontend/javascript/modules/errorReport.js:    this.btn.addEventListener('click', () => this._invokeRavenModal());
frontend/javascript/modules/errorReport.js:  _invokeRavenModal() {
frontend/javascript/modules/errorReport.js:    Raven.showReportDialog({
frontend/javascript/app.js:// Install Raven to send errors to Sentry
frontend/javascript/app.js:import Raven from 'raven-js';
frontend/javascript/app.js:Raven
frontend/javascript/app.js:  Raven.captureException(err.reason);
```

**Comment by mxmeinhold** [comment]
I had to change `/static/js/app.js` to include the full local url in `base_protected.html` to test this.
Sentry get user feedback modal

 As a TODO, we should set that up more automatically.

**Review by adamhb123 — APPROVED** [review]
[LGTM](https://www.csh.rit.edu/~brewer)

**Pull request merged**